### PR TITLE
Add trimpath flag to build args and xgo tool for Go 1.13 release.

### PR DESCRIPTION
**Pull request by novabyte**
This adds support for trimpath to enable platform independent builds with Go 1.13. The docs for the build flag describe the option:

> Remove all file system paths from the resulting executable. Instead of absolute file system paths, the recorded file names will begin with either "go" (for the standard library), or a module path@version (when using modules), or a plain import path (when using GOPATH).

This resolves issue https://github.com/techknowlogick/xgo/issues/37.

### CODE REVIEW

File: xgo.go [hunk 1]
```diff
@@ -31,8 +31,8 @@ func init() {
 		depsCache = filepath.Join(home, ".xgo-cache")
 		return
 	}
-	if user, err := user.Current(); user != nil && err == nil && user.HomeDir != "" {
-		depsCache = filepath.Join(user.HomeDir, ".xgo-cache")
+	if usr, err := user.Current(); usr != nil && err == nil && usr.HomeDir != "" {
```
Comment (kolaente) [inline xgo.go]: Why did you change this?

Comment (techknowlogick) [reply xgo.go]: This is an unrelated change to the PR, but still should be made. 

`user, err := user.Current()` doesn't make sense as `user` would be overridden. changing it to `usr, err := user.Current();` fixes the problem.

Comment (kolaente) [reply xgo.go]: Ah I see, makes sense!

**Comment by techknowlogick** [comment]
Thanks for contribution @novabyte  :)

File: xgo.go (hunk 1 above)
Comment (novabyte) [reply xgo.go]: The current variable name `user` shadows the package import `os/user`. It's bad practice though although a small nit.
